code/LeftAndMain.php

```diff
@@ -1765,7 +1765,7 @@ public function CMSVersionNumber()
             return '';
         }
         $version = $lockModules[$moduleName];
-        if (preg_match('#^([0-9]+)\.([0-9]+)#', $version, $m)) {
+        if (preg_match('#^([0-9]+)\.([0-9]+)#', $version ?? '', $m)) {
```

Comment:
By this point if `$lockModules[$moduleName]` was not null (i.e. `!isset()`) we've already returned out of the method. No need for null coalescing here.